### PR TITLE
streamline the whole documentation

### PR DESCRIPTION
**Pull request by OskarStark**
Refs sonata-project/SonataAdminBundle#3310

### CODE REVIEW

**Comment by rande** [comment]
Conflict

File: Resources/doc/reference/form_types.rst [hunk 1]
```diff
@@ -226,12 +268,24 @@ And the type can now be used:
...
+            ;
+        }
+    }
 
 sonata_type_date_picker and sonata_type_datetime_picker
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------------------------------
 
-Those types integrate `Eonasdan's Bootstrap datetimepicker <https://github.com/Eonasdan/bootstrap-datetimepicker>`_ into a Symfony2 form. They both are available as services, and inherit from ``date`` and ``datetime`` default form types.
+Those types integrate `Eonasdan's Bootstrap datetimepicker`_ into a Symfony2 form.
```
Comment (javiereguiluz) [inline Resources/doc/reference/form_types.rst]: We no longer use `Symfony2`, so this should probably be `Symfony`

File: Resources/doc/reference/form_types.rst [hunk 2]
```diff
@@ -272,36 +327,55 @@ Finally, in your form, you may use the form type as follows:
...
 
 If you look in the classes ``DateTimePickerType.php`` and ``BasePickerType.php`` you can see all the currently available options.
 
 In addition to these standard options, there is also the option `datepicker_use_button` which, when used, will change the widget so that the datepicker icon is not shown and the pop-up datepicker is invoked simply by clicking on the date input.
 
 sonata_type_date_range_picker and sonata_type_datetime_range_picker
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------------------------------------------
 
 `sonata_type_date_range_picker` and `sonata_type_datetime_range_picker` extends the basic range form field types (`sonata_type_date_range` and `sonata_type_datetime_range`).
```
Comment (javiereguiluz) [inline Resources/doc/reference/form_types.rst]: All these single backticks should be double backticks.

Comment (OskarStark) [inline Resources/doc/reference/form_types.rst]: can you explain this to me?

File: Resources/doc/reference/form_types.rst (hunk 1 above)
Comment (OskarStark) [inline Resources/doc/reference/form_types.rst]: done! Thank you!

File: Resources/doc/reference/form_types.rst (hunk 2 above)
Comment (javiereguiluz) [inline Resources/doc/reference/form_types.rst]:
```
`sonata_type_date_range_picker`
```

should be:

```
``sonata_type_date_range_picker``
```

Same for the rest.

**Comment by OskarStark** [comment]
@sonata-project/contributors if travis is green, feel free to push the merge button!

Thank you for your feedback @javiereguiluz

**Pull request merged**